osu.Game/Users/UserActivity.cs

```diff
@@ -52,7 +52,7 @@ public SoloGame(BeatmapInfo info, Rulesets.RulesetInfo ruleset)
                 Ruleset = ruleset;
             }
 
-            public override string Status => @"Playing alone";
+            public override string Status => Beatmap.Ruleset.CreateInstance().PlayingVerb;
```

Comment:
why using `Beatmap`'s when you already have the `Ruleset`?

Reply:
`RulesetInfo` doesn't have the member `PlayingVerb`

Maybe the names should be changed, i.e change `info` to `mapInfo` and `ruleset` to `rulesetInfo` to make it clearer?

Reply:
That's not what i'm talking about, simply
```suggestion
            public override string Status => Ruleset.CreateInstance().PlayingVerb;
```

Reply:
my bad, misread your comment.